src/platform/web/Platform.js

```diff
@@ -307,6 +312,32 @@ export class Platform {
         return DEFINE_VERSION;
     }
 
+    get themeLoader() {
+        return this._themeLoader;
+    }
+
+    get preferredColorScheme() {
```

Comment:
Can we move this to the ThemeLoader? It's crowded here already, and this seems more or less related to theme loading ...

Reply:
It uses `window.` which is platform dependent?